src/main/webapp/skin.xml

```diff
@@ -23,6 +23,8 @@
     xsi:schemaLocation="http://www.jasig.org/uportal/web/skin https://source.jasig.org/schemas/resource-server/skin-configuration/skin-configuration-v1.2.xsd">
 
 
+    <css>css/newsreader.css</css>
```

Comment:
Looks like we can't add a css file to any of the *skin.xml files that is itself generated by the build, so we'll go with the per-jsp approach.